.prettierrc

```diff
@@ -0,0 +1,29 @@
+{
+    "debug": false,
+    "auto_format_on_save": true,
+    "auto_format_on_save_requires_prettier_config": true,
+    "auto_format_on_save_excludes": [
+        "*/node_modules/*",
+        "*.json"
+    ],
+    "allow_inline_formatting": false,
+    "use_local_babel": true,
+    "prettier_options": {
+        "arrowParens": "avoid",
+        "bracketSpacing": true,
+        "jsxBracketSameLine": false,
+        "parser": "babylon",
+        "printWidth": 110,
+        "semi": true,
+        "tabWidth": 4,
+        "singleQuote": true,
+        "trailingComma": "es5",
+        "useTabs": false
+    }
+}
+
+
+
+
+
+
```

Comment:
make sure to add a prettier rule about not to have multiple blank lines at the end of the file